gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -32,6 +32,15 @@
 _MAX_BQ_NUM_PARTITIONS = 4000
 _TOTAL_BASE_PAIRS_SIG_DIGITS = 4
 _PARTITION_SIZE_SIG_DIGITS = 1
+TABLE_SUFFIX_SEPARATOR = '__'
+SAMPLE_TABLE_SUFFIX_SEPARATOR = '_'
```

Comment:
We only use these two consts in this module (and one test module). Consider adding `_` to them.

Reply:
Applied this change in #554 . Reverting this PR.